AUTHORS.md

```diff
@@ -103,5 +103,12 @@ the following authors:
             <sub><b>LeoFonsecaP</b></sub>
         </a>
     </td></tr>
+    <td align="center">
+        <a href="https://github.com/LeoFonsecaP">
+            <img src="https://avatars.githubusercontent.com/u/63270818?v=4" width="100;" alt="Henrique1792"/>
+            <br />
+            <sub><b>Henrique1792</b></sub>
+        </a>
+    </td></tr>
```

Comment:
I think the profile link is incorrect. Aside from that, there's no need to manually edit this action, GitHub Actions does that automatically since #37. I don't know if manual changes can cause any conflicts with the existing workflow.

Reply:
Oh, I wasn't aware about #37 - my bad 😬 - I'll apply those suggestions!